lib/citriage.rb

```diff
@@ -74,39 +79,70 @@ def list_jobs json
       mod_status = true
       failed_job = String.new
       begin
-        json['jobs'].each do |job|
-          if job['color'] == 'red'
-            mod_status = false
-            failed_job = job['url']
-            break
-          else
-            mod_status = true
+        json.each do |name, branch|
```

Comment:
Oh! Yeah that is definitely a bug. My intention was for the status to appear red if either master or stable/release is red.

My reasoning for that is that a) it's consistent with the view you see on Jenkins (if any branch is red the pipeline will be red) and b) I missed some errors on catalog_preview stable for a few days because it appeared green with the triage gem (which is what prompted me to do this work).

That's my thinking on it but I could go either way, especially with the --verbose view. I can either fix the bug and have it reflect both branches as intended, or change it back to how it was before (reflecting only master).

Let me know what you think is best!